cmd/manager/main.go

```diff
@@ -198,6 +198,10 @@ func main() {
 		exit(err)
 	}
 
+	err = defaults.RemoveObsoleteOpsrc(clientGo)
```

Comment:
Won't this only clean things up on restart? Do we want to make this part of the active defaults logic instead?

Reply:
@kevinrizza that is a good point. What do you mean by `active defaults logic` though?

Reply:
Can't we only switch an operator source to a catalogsource after a restart? (because the definition is in the pod, the pod will need to be updated)